src/components/LeftNavBar.css

```diff
@@ -0,0 +1,7 @@
+.contentNav{
```

Comment:
Would be nice to keep it in tailwind only. If you really feel a class is needed, you can use tailwind config to put this in a class that is configured on tailwind's configuration level, so that you can now use this class anywhere you want.

Much more likely is it the case that you did this because you couldn't get the tailwind way to do scrollbar-gutter (the others seem like things that tailwind already has). If that's the case in your `tailwind.config.js`

```
module.exports = {
  theme: {
    extend: {
      scrollbarGutter: {
        'stable': 'stable',
      },
      overflowBehaviorY: {
        'auto': 'auto',
      }
    },
  },
  variants: {},
  plugins: [],
}
```
and then you can use it anywhere you want now like 

```
<div className="block h-[calc(50vh)] overflow-auto overflow-behavior-y-auto scrollbar-gutter-stable">
```

Some custom utilities related info:
https://tailwindcss.com/docs/overflow

Reply:
Transferred Scrollbar css to tailwind.config.js file for reusability, thanks @launchupio for the  giga-brain 🧠 suggestion 🔥